polkadot/runtime/common/src/slots/mod.rs

```diff
@@ -86,6 +93,10 @@ pub mod pallet {
 		#[pallet::constant]
 		type LeasePeriod: Get<BlockNumberFor<Self>>;
 
+		/// The number of blocks in past upto which early refund is allowed.
+		#[pallet::constant]
+		type EarliestRefundPeriod: Get<BlockNumberFor<Self>>;
```

Comment:
I do find the name a little confusing. When I was first reading the PR description, I assumed this would be number of _lease periods_.
```suggestion
		/// The number of blocks prior to lease expiration, after which a parachain can refund its
		/// bond prematurely. A value of zero means that funds cannot be refunded until the lease
		/// expires.
		#[pallet::constant]
		type EarliestRefundPeriod: Get<BlockNumberFor<Self>>;
```